products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -12,32 +12,75 @@
+For a multi-repo search MVP, we are adding an HTML text input and [Algolia's DocSearch JS snippet](https://github.com/algolia/docsearch) to the [`va.gov-team` GitHub Pages site](https://department-of-veterans-affairs.github.io/va.gov-team/). This will use [Algolia's Search API](https://www.algolia.com/products/search/) to query Algolia's hosted database (https://www.algolia.com/pricing/), which will be populated by [Algolia's scraper](https://github.com/algolia/docsearch-scraper) using a [configuration we supply](docs/docsearch-scraper-config.json).
+
+In general, this kind of system requires the following components:
+
+- Page with Input Textbox - We need a public page with a search input where users can input their search term(s).
+- Index/Database - We need a place to store the records that will be returned to the user.
+- Search API - We need a way to get data out of the index.
+- Crawler/Scraper - We need a crawler to scan our documentation sources and push data to our index.
+- Pipeline - We need a way to routinely run the crawler
+
+By using Algolia's Search API product with their open source Dropdown Search-UI and open source scraper, we can avoid building/maintaining the database, API, UI, and crawler components.
+
+That means we'd be responsible for:
+
+- Converting `va.gov-team`'s `docs/index.md` from markdown to HTML
+- Adding a search input to `va.gov-team`'s `docs/index.html`
+- Adding the DocSearch `<style>` and `<script>` tags to `va.gov-team`'s `docs/index.html`
+- Writing the config file for the scraper
+- Adding CircleCI to `va.gov-team`
+- Adding a scheduled job to `va.gov-team`'s CircleCI pipeline to run the scraper once every 24 hours
+- Manually triggering the initial scraper run to populate our Algolia index
+
+#### Usage limits
+
+[Algolia's pricing page](https://www.algolia.com/pricing/) shows three different tiers, in addition to an enterprise plan. The primary differentiator between the tiers is the number of records and the number of operations (read/write/edit/delete) included with each plan.
+
+##### The Community Plan
+
+The free tier, the Community plan, includes up to 10K records and 50K operations (read/write/edit/delete). If we exceed the limits of the free tier, we'd have to upgrade to a paid tier.
+
+##### The Starter Plan
+
+The middle tier, the Starter plan, includes up to 50K records and 250k operations. If we exceed the limits of the Starter plan, it would cost an extra $10/month for every additional 100K operations, and an extra $10/month for every additional 20K records.
+
+##### The Pro Plan
 
-A successful implementation will have these components.
+The upper tier, the Pro plan, includes up to 1M records and 5M operations. If we exceed the limits of the Pro plan, it would cost an extra $5/month foe every 100K additional operations, and an extra $5/month for every additional 20K records.
```

Comment:
```suggestion
The upper tier, the Pro plan, includes up to 1M records and 5M operations. If we exceed the limits of the Pro plan, it would cost an extra $5/month for every 100K additional operations, and an extra $5/month for every additional 20K records.
```